react-app-rewired/headers/csps/fiatRamps/onRamper.ts

```diff
@@ -1,5 +1,7 @@
 import type { Csp } from '../../types'
 
 export const csp: Csp = {
+  'frame-src': ['https://widget.onramper.com', 'https://swap.onramper.com', 'https://buy.moonpay.com'],
+  'frame-ancestors': ['*'],
```

Comment:
ok, spec doesn't allow `frame-ancestors` in meta tags https://content-security-policy.com/frame-ancestors/